andela/ah-backend-thor
# #161382389 Users should receive a JWT on successful Registration

#### What does this PR do?
Users can recieve JWT tokens upon login
#### Description of Task to be completed?
when user logs in via `api/users/login/`
users recieves a `token` which then can use to access protected features
#### How should this be manually tested?
run server with `python manage.py runserver`
in `postman` sign up a user with object of this format


          { "user": {
                "username": "kegz",
                "email": "kegz@gmail.com",
                "password": "1231234567"
            }
        }


- then login the user with object of format

          { "user": {
                "email": "kegz@gmail.com",
                "password": "1231234567"
            }
        }

- one should also use the following headers

`Content-Type` ==> `Application/Json`
`Authorization` ==> `Token ` + `actual token`

#### Any background context you want to provide?
to test this, one `should` have `PostgreSQL` installed locally and should clone  the repository with

`git clone https://github.com/andela/ah-backend-thor.git`

then create a database called `ah-backend`

make and run migrations
`python manage.py makemigrations`
`python manage.py migrate`

#### What are the relevant pivotal tracker stories?
#161382389
